### DISCUSSION TITLE
配置文件

### DISCUSSION THREAD

**Discussion by gufeng0** [tamlok/vnote#1937] [Q&A]
请问支持自定义配置文件路径吗？ macOS系统，快捷键该如何修改，比如 ActivateNextTab 快捷键改为 Command + h

**Comment by tamlok** [comment]
1. 配置文件会优先从app所在目录读取，无法自定义。具体参考https://vnotex.github.io/vnote/zh_cn/#!docs/%E7%94%A8%E6%88%B7/%E9%85%8D%E7%BD%AE.md#toc_2
2. 快捷键可以修改`vnotex.json`文件里面对应的条目，`Command+H`应该是`Ctrl+H`即可。
